open_rfps/Decentralized Forum RFP

```diff
@@ -0,0 +1,65 @@
+buy and sell digital and non-digital goods using a contract system(at their own risk).
+The XOR will be used for all the payments in the forum (like subscriptions, contracts, paid stickers, etc).
+To run the forum, me and my team will have to register a company, so we can pay all the people who will work for the project legally (coders, UI designers, mods, etc.), pay for
+the servers that will host the forum and also probably we will use some of the funds for marketing. I will write the proposed budget below.
+
+**Scope of Work (describe the scope of work in as much detail as possible)**
+All the technology that we may need to code the forum is available. Polkadot.js wallet seems to be compatible with whatever we have in our mind as team. 
+Would be great if we could use wallet connect for Fearless Wallet, Sora DAE and other mobile wallets. This may happen in the future.
+
+**Current roadblocks**
+None 
+
+**Evaluation metrics and criteria**
+- People will be able to post comments and threads by only signing messages with their wallets.
+
+- There will be sections such as Economy, Life, Code, Games, Tech, Science, Market, Cryptocurrencies, and many others.
+
+- To prevent spam posts/threads, it will be required each user to hold at least 40-50$ worth of XOR in their address (the equivalent amount of $ may change).
+
+- There will be an exclusive Sora and Polkaswap section in the forum so all the users can read about new updates and learn more about the network.
+
+- We will create a contract system for the Marketplace section, people will give a score for the quality of the products/services of the vendors after every deal.
+
+- The contracts will be paid in XOR, they'll be off-chain contracts in a server, in the future we may upgrade the contract system and they may be on-chain contracts to the public Sora's blockchain.
+
+- There will be groups owned by influencers/youtubers in order to bring more users, people will pay for subscriptions the same way they pay at other platforms, so they have access to each group.
+
+- Every group will have its section in the forum and only people who paid subscriptions will have access.
+
+- Subscriptions, group sales, contracts, stickers, and in general everything in the forum will be paid only with XOR.
+
+- We will be able to blacklist an address if they violated the T.O.S.
+
+- Sora Parliament will always be helpful with their new ideas for the forum.
+
+- There will be various changes to the UI as the time goes by based on what community asks/likes.
+
+- Some of the funds will be used to protect the servers from DDOS attacks, hopefully no additional resources will be needed.
+
+**Milestones** 
+Project to begin March 2022, be ready for testing at latest by July 2022, and released to the public once it's fully functional.
+
+**Deliverables**
+- A fully functional forum that uses XOR as its native currency.
```

Comment:
fully functional has a very wide meaning. Please be more concrete.

Reply:
What I meant is that we will deliver a forum that every section/button will work as it should be and that whatever we proposed (stickers, groups sales, subscriptions, everything will be paid only with XOR, including the 40-50$ worth of XOR per address to prevent spam/bots) will be implemented.